apps/core/contexts/searchContext/helpers.js

```diff
@@ -26,7 +27,7 @@ const PARSER_CONFIG = {
   isNightsInDestinationSelected: 'boolean',
 };
 
-export function parseURLqueryToState(query: Object) {
+export function parseURLqueryToState(query: any) {
```

Comment:
Is that change from `Object` to `any` necessary? Not that `Object` is that much better. But if we cannot use some more exact type it's probably ok.

Reply:
Good point, I try to handle the flow and forgotten to change it back.

Reply:
done